src/vl-map.js

```diff
@@ -65,7 +65,8 @@ export class VlMap extends VlElement(HTMLElement) {
     get _projection() {
         return new ol.proj.Projection({
             code: 'EPSG:31370',
-            extent: this._extent
+            extent: this._extent,
+            getPointResolution: (r) => r
```

Comment:
@driesva enig idee waarom dit nodig was, dit werd voorlopig niet overgenomen maar er zijn geen testen die falen dus ben niet zeker of deze wijziging nodig is.

Reply:
In mijn comment had ik "fix issue with ScaleLine". Ik herinner mij nog vaag iets dat de schaal niet klopte...

Dus best eens even nakijken. 

Misschien relevant:
https://github.com/openlayers/openlayers/issues/7556

Reply:
Inderdaad, eenheid lijkt verkeerd te zijn bij de scaler. Ik zal het toepassen en kijken of ik hiervoor ook een test kan toevoegen.